babel.config.js

```diff
@@ -25,99 +25,394 @@
 
 'use strict';
 
-const minimist = require('minimist');
-const {isTravisBuild} = require('./build-system/common/travis');
-const argv = minimist(process.argv.slice(2));
+const argv = require('minimist')(process.argv.slice(2));
+const experimentsConfig = require('./build-system/global-configs/experiments-config.json');
+const experimentsConstantBackup = require('./build-system/global-configs/experiments-const.json');
+const fs = require('fs');
 
-const isClosureCompiler =
-  argv._.includes('dist') ||
-  argv._.includes('check-types') ||
-  (argv._.length == 0 && argv.compiled);
-const {esm} = argv;
+/**
+ * Default options for transform-react-jsx. Used by pre-closure and dep-check.
+ */
+const defaultJsxOpts = {
+  pragma: 'Preact.createElement',
+  pragmaFrag: 'Preact.Fragment',
+  useSpread: true,
+};
+
+/**
+ * Default preset-env plugin. Used by dep-check and tests.
+ */
+const defaultPresetEnvPlugin = [
+  '@babel/preset-env',
+  {
+    bugfixes: true,
+    modules: 'commonjs',
+    loose: true,
+    targets: {'browsers': ['Last 2 versions']},
+  },
+];
+
+/**
+ * Default filter-imports plugin. Used by pre-closure.
+ */
+const defaultFilterImportsPlugin = [
+  'filter-imports',
+  {
+    imports: {
+      './polyfills/document-contains': ['installDocContains'],
+      './polyfills/domtokenlist': ['installDOMTokenList'],
+      './polyfills/fetch': ['installFetch'],
+      './polyfills/math-sign': ['installMathSign'],
+      './polyfills/object-assign': ['installObjectAssign'],
+      './polyfills/object-values': ['installObjectValues'],
+      './polyfills/promise': ['installPromise'],
+      './polyfills/array-includes': ['installArrayIncludes'],
+      '../third_party/css-escape/css-escape': ['cssEscape'],
+      '../build/ampshared.css': ['cssText', 'ampSharedCss'],
```

Comment:
These were previously in a different object since they noted that a valid transformed document was required to use them.

Perhaps a comment to indicate the same here for items previously in `validTransformedRemovableImports`?